packages/ui/tailwind-utils-config/components/progress.ts

```diff
@@ -147,11 +149,11 @@ export default {
       },
       '&::-webkit-progress-value': {
         border: 'none',
-        'background-color': 'var(--cn-comp-slider-track-progress)'
+        'background-color': 'var(--cn-progress-bar-bg-color)'
```

Comment:
can we use `--cn-comp-slider-track-progress` directly?

Reply:
We used before. But in cases when we need to change the color of the progress bar, we had to use very complex selectors like this one:
`[&_.cn-progress-root::-moz-progress-bar]:bg-cn-background-8 [&_.cn-progress-root::-webkit-progress-value]:bg-cn-background-8`

When the color is applied through --cn-comp-slider-track-progress, it allows us to easily change it via style prop:
```js
style={{ '--cn-progress-bar-bg-color': 'var(--cn-set-gray-soft-bg)' } as CSSProperties}
```